pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -308,24 +308,26 @@ def newpkgcb(pkgId, name, arch):
                         for update in updates:
                             update_record = UpdateRecord(**UpdateRecord.createrepo_to_dict(update))
                             update_record.digest = RpmFirstStage.hash_update_record(update)
+                            future_relations = {'collections': {}, 'references': []}
 
                             for collection in update.collections:
                                 coll_dict = UpdateCollection.createrepo_to_dict(collection)
                                 coll = UpdateCollection(**coll_dict)
+                                future_relations['collections'][coll] = []
```

Comment:
This looks like a use caseof `collections.defaultdict` but I think it's fine how it's written here. No change requested.